src/opencl_pbkdf2_hmac_sha1.h

```diff
@@ -26,12 +28,28 @@ typedef struct {
 	unsigned int dk[((OUTLEN + 19) / 20) * 20 / sizeof(unsigned int)];
 } pbkdf2_out;
 
+//#if !defined (OPENCL_PBKDF2_HMAC_SHA1_2_LIMB) && !defined (OPENCL_PBKDF2_HMAC_SHA1_3_LIMB)
+//typedef struct {
+//	unsigned int  length;
+//	unsigned int  outlen;
+//	unsigned int  iterations;
+//	unsigned char salt[55];
+//} pbkdf2_salt;
+//#elif defined (OPENCL_PBKDF2_HMAC_SHA1_2_LIMB)
+//typedef struct {
+//	unsigned int  length;
+//	unsigned int  outlen;
+//	unsigned int  iterations;
+//	unsigned char salt[115];
+//} pbkdf2_salt;
+//#elif defined (OPENCL_PBKDF2_HMAC_SHA1_3_LIMB)
```

Comment:
Perhaps you should drop the commented-out stuff? Or is it better left there?

Reply:
I left it there, as a 'guide'.  I think we might see some performance gains (possibly not), by having the formats define that they only need 51 byte salt, (or some other size).

We can take it out for now, if you like, but I would rather the comments be there, since that part of the code is 'done', then later we can add the defines in the proper places within the formats.

But since this is only the salt size, it may not be that big of an impact on the formats at all. I think the only time there is any data transfer to GPU is when the salt is set.

Reply:
Lets keep it. I'll look into some clean-up and optimizations later, letting GPU side know actual max-size of salt and adopt at compile time. This is not in the hot loop so we'll probably not see much gain, but anyway.